spec/lib/alerts/de_userfying_edit_alert_monitor_spec.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require "#{Rails.root}/lib/alerts/de_userfying_edit_alert_monitor"
+
+def mock_mailer
+  OpenStruct.new(deliver_now: true)
+end
+
+describe DeUserfyingEditAlertMonitor do
```

Comment:
In addition to this spec, it would be very useful to add a preview for this alert (in `alert_mailer_preview.rb`) so that we can easily see what an email for this alert type would look like.

Reply:
Done. There was also a mistake in the class name.